common/views/pages/_app.tsx

```diff
@@ -0,0 +1,350 @@
+  useEffect(() => {
+    // Prismic preview and validation warnings
+    if (document.cookie.match('isPreview=true')) {
+      window.prismic = {
+        endpoint: 'https://wellcomecollection.prismic.io/api/v2',
+      };
+      const prismicScript = document.createElement('script');
+      prismicScript.src = '//static.cdn.prismic.io/prismic.min.js';
+      document.head && document.head.appendChild(prismicScript);
+      (function() {
+        const validationBar = document.createElement('div');
+        validationBar.style.position = 'fixed';
+        validationBar.style.width = '375px';
+        validationBar.style.padding = '15px';
+        validationBar.style.background = '#e01b2f';
+        validationBar.style.color = '#ffffff';
+        validationBar.style.bottom = '0';
+        validationBar.style.right = '0';
+        validationBar.style.fontSize = '12px';
+        validationBar.style.zIndex = '2147483000';
+
+        const validationFails: string[] = [];
+
+        const descriptionEl = document.querySelector(
+          'meta[name="description"]'
+        );
+        if (descriptionEl && !descriptionEl.getAttribute('content')) {
+          validationFails.push(`
+            <b>Warning:</b>
+            This piece of content is missing its description.
+            This helps with search engine results and sharing on social channels.
+            (If this is from Prismic, it's the promo text).
+          `);
+        }
+
+        const imageEl = document.querySelector('meta[property="og:image"]');
+        if (imageEl && !imageEl.getAttribute('content')) {
+          validationFails.push(`
+            <b>Warning:</b>
+            This piece of content is missing its promo image.
+            This is the image that will be shown across our site,
+            as well as on social media.
+          `);
+        }
+
+        if (validationFails.length > 0) {
+          validationFails.forEach(function(validationFail) {
+            const div = document.createElement('div');
+            div.style.marginBottom = '6px';
+            div.innerHTML = validationFail;
+            validationBar.appendChild(div);
+          });
+          document.body && document.body.appendChild(validationBar);
+        }
+      })();
+    }
+  }, []);
+
+  // sentry
+  useEffect(() => {
+    Raven.config('https://f756b8d4b492473782987a054aa9a347@sentry.io/133634', {
+      shouldSendCallback() {
+        const oldSafari = /^.*Version\/[0-8].*Safari.*$/;
+        const bingPreview = /^.*BingPreview.*$/;
+
+        return ![oldSafari, bingPreview].some(r =>
+          r.test(window.navigator.userAgent)
+        );
+      },
+      whitelistUrls: [/wellcomecollection\.org/],
+      ignoreErrors: [
+        /Blocked a frame with origin/,
+        /document\.getElementsByClassName\.ToString/, // https://github.com/SamsungInternet/support/issues/56
+      ],
+    }).install();
+  }, []);
+
+  return (
+    <>
+      <Head>
```

Comment:
It does feel as though all of this, including the hooks should move to the `PageLayout`. If anything it would make it way easier to test.